src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Resources.xaml.cs

```diff
@@ -13,7 +17,10 @@ public SharedResources()
         private void PackageIconImage_ImageFailed(object sender, ExceptionRoutedEventArgs e)
         {
             var image = sender as Image;
-            image.Source = Images.DefaultPackageIcon;
+
+            e.Handled = true; // don't repropagate the event
+            var binding = BindingOperations.GetBindingExpression(image, Image.SourceProperty);
+            binding.UpdateTarget();
```

Comment:
[`BindingExpression.UpdateTarget()`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.bindingexpression.updatetarget?view=netframework-4.8#exceptions) can throw if "[t]he binding has been detached from its target."  Is this a possibility we should guard against?